astroquery/eso/core.py

```diff
@@ -207,6 +207,8 @@ def list_instruments(self, cache=True):
                     if instrument not in self._instrument_list:
                         self._instrument_list.append(instrument)
             self._instrument_list.append(u'harps')
+            self._instrument_list.append(u'ambient_paranal')
+            self._instrument_list.append(u'meteo_paranal')
```

Comment:
Ohh, I meant that I don't see it either, but maybe the logic that it's an outlier among the VLT/Paranal (since VIRCAM is there) instruments. My simple guess they simply forgot about La Silla for this page...